### PR TITLE
[aptos] genesis readme

### PR DESCRIPTION
**Pull request by davidiw**
We can make a variance of this for the actual testnet process.

### CODE REVIEW

File: crates/aptos/README.md [hunk 1]
```diff
@@ -581,3 +581,130 @@ Additionally, profiles can replace addresses in the function id.
...
+* A unique `ChainId` (u8) that distinguishes this from other deployments
+* For test chains, there also exists an account that manages the minting of TestCoin
+
+## Generating Genesis
+
+* The genesis organizer constructs a `Layout` and distributes it.
+* The genesis organizer prepares the Aptos framework's bytecode and distributes it.
+* Each participant generates their `ValidatorConfiguration` and distributes it.
+* Each participant generates a `genesis.blob` from the resulting contributions
+* The genesis organizer executes the `genesis.blob` to derive the initial waypoint and distributes it.
```
Comment (sherry-x) [inline crates/aptos/README.md:600]: waypoint can be generated by participant too, maybe here we say organizer will publish genesis.blob and waypoint, participant will validate they got same result? (e.g. sha256)

**Review by sherry-x — APPROVED** [review]

File: crates/aptos/README.md (hunk 1 above)
Comment (davidiw) [reply crates/aptos/README.md:600]: Should discuss this -- if our viewpoint is that the waypoint is going to be used by participants from the one they generate, we might as well make the waypoint optional and reduce operational burden.

Comment (sherry-x) [reply crates/aptos/README.md:600]: well right now genesis.blob and waypoint are generate in one command, so they would have both anyway

**Pull request closed without merging**